base/float.jl

```diff
@@ -767,74 +767,74 @@ realmin() = realmin(Float64)
 realmax() = realmax(Float64)
 
 """
-    eps(::Type{T}) where T<:AbstractFloat
-    eps()
+    ulp(::Type{T}) where T<:AbstractFloat
+    ulp()
 
 Return the *machine epsilon* of the floating point type `T` (`T = Float64` by
```

Comment:
Shouldn't "machine epsilon" be updated (or adjusted to also mention ULP) here as well?